osi_object.proto

```diff
@@ -950,5 +954,64 @@ message MovingObject
                 BRAKE_LIGHT_STATE_STRONG = 4;
             }
         }
+
+        // Definition of vehicle roles. 
+        //
+        // The role specifies the perceived role of a vehicle. Therfore, a 
+        // civil police car is set to ROLE_CIVIL as long as it cannot be 
+        // perceived as a ROLE_POLICE. 
+        // The normal use case will consist of 99% ROLE_CIVIL, as this role is 
+        // used for normal vehicles.
+        // To check whether a role is "active" (e.g. blue flashing light of an 
+        // ambulance or an amber flashing light of a sweeper) can be looked up 
+        // in message LightState.
+        // 
+        //
+        enum Role
+        {
+            // Role of vehicle is unknown (must not be used in ground truth).
+            //
+            ROLE_UNKNOWN = 0;
+
+            // Other (unspecified but known) role of vehicle.
+            //
+            ROLE_OTHER = 1;
+
+            // The vehicle role is ambulance.
+            //
+            ROLE_AMBULANCE = 2;
```

Comment:
OSI CCB 23.05. 
- please change order of Ambulance and Civil. The standard case should be the first one.

Reply:
Change is applied.